dist/jquery.popup-interactive-world-map.coffee

```diff
@@ -0,0 +1,49 @@
+$ ($, window, document) ->
+  pluginName = 'interactiveWorldMapCountryPopup'
```

Comment:
The name and the filename doesn't match up. Was an oversight of me, sorry. Rename the file to match the plugin name, plase.